moss-client/moss-client-common/pom.xml

```diff
@@ -70,6 +70,40 @@
             <artifactId>jackson-databind</artifactId>
             <version>2.9.9</version>
         </dependency>
+        <dependency>
```

Comment:
The mocking and deeptestutils dependencies can be removed.
It looks like you only need to add the JUnit dependency.